__tests__/app.test.js

```diff
@@ -225,3 +225,21 @@ describe('PATCH /api/articles/:article_id', () => {
     });
   })
 })
+
+describe('DELETE /api/comments/:comment_id', () => {
+  test('responds 204 status when deleted comment', () => {
+    return request(app)
+    .delete("/api/comments/3")
+    .expect(204)
+  })
+
+  test('responds with a 404 error when passed an invalid id', () => {
```

Comment:
lets reword this description to something like 'responds with a 404 error when passed a valid id that doesn't exist'